app/src/main/java/io/gnosis/safe/ui/transactions/details/TransactionDetailsFragment.kt

```diff
@@ -141,6 +142,11 @@ class TransactionDetailsFragment : BaseViewBindingFragment<FragmentTransactionDe
             }
         }
 
+        binding.advanced.setOnClickListener {
+            //TODO: pass arguments
```

Comment:
This is where we wire @biafra23 changes into yours?

Reply:
yes